frontend/src/pages/products.js

```diff
@@ -1,3 +1,43 @@
+import { useNavigate } from 'react-router';
+
 export default function Product() {
-  return <div data-testid='products'>Checkout our products :D</div>;
+  const navigate = useNavigate();
+  const products = [
+    {
+      id: 1,
+      imageUrl:
+        'https://images.pexels.com/photos/585753/pexels-photo-585753.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500',
+      description: 'A cup fit for a software engineer.',
+      price: 4.99,
+    },
+  ];
+
+  return (
+    <div data-testid='products'>
+      <div className=' bg-gradient-to-r from-blue-800 via-blue-500 to-blue-800 p-5 text-center text-4xl'>
+        <button onClick={() => navigate('/')} className='blue-500'>
```

Comment:
long-term I think we will have a component that we use in home that displays the same data, but this is great progress